kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -77,8 +78,40 @@ function showExploreContent(store, id) {
 }
 
 function showLearnRoot(store) {
+  store.dispatch('SET_LOADING');
   store.dispatch('SET_PAGE_NAME', PageNames.LEARN_ROOT);
-  store.dispatch('SET_PAGE_STATE', {}); // TODO
+
+  Resources.getCollection({ recommendations: '' }).fetch()
+    .then((children) => {
```

Comment:
I think currently recommendations will return random nodes, not necessary the children, the naming might mislead people